webpack.config.cjs

```diff
@@ -253,7 +253,11 @@ const storyEmbedBlock = {
 const webStoriesBlock = {
   ...sharedConfig,
   entry: {
-    'web-stories-block': './assets/src/web-stories-block/index.js',
+    'web-stories-block': [
+      './assets/src/web-stories-block/index.js',
+      './assets/src/web-stories-block/block/edit.css',
+    ],
+    'web-stories-list-styles': './assets/src/web-stories-block/css/style.css',
```

Comment:
What does the change here do?

Reply:
This will create two separate style sheets one for the block ( editor ) and one for the front-end ( web-stories-list-styles ).

With the previous set-up it was working fine but here since we also need some styles for the editor it started creating issues. This set-up is typically used for projects where we don't import styles in JS ( import './some-style' ) and works perfectly for our use case.